### DISCUSSION TITLE
Some apps find that I use a proxy and refuses to connect to the Internet

### DISCUSSION THREAD

**Discussion by bin840004** [mitmproxy/mitmproxy#4945] [Intercept Everything!]
Some apps can detect that the user's WiFi has set up a proxy and then refuse to connect to the Internet.It seems that using a transparent proxy can make the app unaware that the traffic passes through mitmproxy and does not need to set up a proxy, but Windows 10 does not support transparent proxy

**Comment by dufferzafar** [comment]
I haven't personally used mitm on a Windows machine so no clue what would happen there.

But this does sound interesting, could you please provide an example of the app that can detect the proxy?

**Comment by yodaluca23** [comment]
I have tried with normal mode proxy, and using wireguard (transparent) proxy, both the iOS app MyQ is able to detect, and it refuses to load or connect, giving the error "There was an error connecting to the server. Please check your connection and try again."

**Comment by rosydawn6** [comment]
This issue is also discussed here: https://github.com/mitmproxy/mitmproxy/discussions/4945 
Some sites are now using techniques to detect proxy and vpn connections.
